### DISCUSSION TITLE
Command line screen showing some junk characters in between text and my output directory pointing german resources

### DISCUSSION THREAD

**Discussion by nagulandev** [shibayan/Sharprompt#253] [Q&A]
using sharprompt in my project. 
The command line screen has some junk characters and also cursor position is misplaced.
Why my output directory pointing to german resources

Help to get this issue resolved

**Comment by shibayan** [comment]
There may be a problem with the character encoding of the terminal; if you are using non-ASCII characters, make sure they are in a supported code such as UTF-8.
